src/core/qgsproperty.h

```diff
@@ -124,7 +117,7 @@ class CORE_EXPORT QgsPropertyDefinition
      * \param type one of the predefined standard property template
      * \param origin The origin of the property
      */
-    QgsPropertyDefinition( const QString &name, const QString &description, StandardPropertyTemplate type, Origin origin = QgsPropertyDefinition::Pal );
+    QgsPropertyDefinition( const QString &name, const QString &description, StandardPropertyTemplate type, const QString &origin = QStringLiteral( "labeling" ) );
```

Comment:
origin should default to QString(), not labeling.

Also note that we had issues in the past when using QStringLiteral in header files, so better to avoid that.

Reply:
done